src/aircraft_cmd.cpp

```diff
@@ -1782,6 +1791,11 @@ static void AirportClearBlock(const Aircraft *v, const AirportFTAClass *apc)
 	if (apc->layout[v->previous_pos].blocks != apc->layout[v->pos].blocks) {
 		Station *st = Station::Get(v->targetairport);
 
+		if (st->airport.blocks.Test(AirportBlock::Zeppeliner) &&
+				(apc->layout[v->previous_pos].blocks == AirportBlock::RunwayIn)) {
```

Comment:
```suggestion
				apc->layout[v->previous_pos].blocks == AirportBlock::RunwayIn) {
```
The extra parentheses are not needed.